x-pack/plugins/ml/public/application/explorer/alerts_panel.tsx

```diff
@@ -0,0 +1,152 @@
+  const durationFormatter = useFieldFormatter(FIELD_FORMAT_IDS.DURATION);
+
+  const { anomalyTimelineStateService, anomalyDetectionAlertsStateService } =
+    useAnomalyExplorerContext();
+
+  const alertsData = useObservable(anomalyDetectionAlertsStateService.anomalyDetectionAlerts$);
+  const selectedAlertsData = useObservable(anomalyDetectionAlertsStateService.selectedAlerts$);
+  const countByStatus = useObservable(anomalyDetectionAlertsStateService.countByStatus$);
+
+  const annotationXDomain = useObservable(anomalyTimelineStateService.timeDomain$);
+
+  const swimlaneContainerWidth = useObservable(
+    anomalyTimelineStateService.getContainerWidth$(),
+    anomalyTimelineStateService.getContainerWidth()
+  );
+
+  const columns: Array<EuiBasicTableColumn<AnomalyDetectionAlert>> = [
+    {
+      field: 'ruleName',
+      name: 'Rule name',
+      sortable: true,
+      truncateText: false,
+    },
+    {
+      field: 'jobId',
+      name: 'Job ID',
+      sortable: true,
+      truncateText: false,
+    },
+    {
+      field: 'anomalyTimestamp',
+      name: 'Anomaly time',
+      sortable: true,
+      truncateText: false,
+      render: (value: number) => dateFormatter(value),
+    },
+    {
+      field: 'timestamp',
+      name: 'Triggered at',
+      sortable: true,
+      truncateText: false,
+      render: (value: number) => dateFormatter(value),
+    },
+    {
+      name: 'Duration',
+      truncateText: false,
+      render: (value: AnomalyDetectionAlert) =>
+        durationFormatter(value.end_timestamp - value.timestamp),
+    },
+  ];
+
+  const yValueMap: Record<string, number> = {};
+  const chartData = alertsData
+    ?.map((alert, i) => {
+      if (!yValueMap.hasOwnProperty(alert.ruleName)) {
+        yValueMap[alert.ruleName] = Math.max(...Object.values(yValueMap), 0) + 1;
+      }
+      const yV = yValueMap[alert.ruleName];
+      return [
+        // null is required to make isolated points
+        [alert.timestamp, null, alert.ruleName],
+        [alert.timestamp, yV, alert.ruleName],
+        [alert.end_timestamp, yV, alert.ruleName],
+      ];
+    })
+    .flat();
+
+  return (
+    <>
+      <CollapsiblePanel
+        isOpen={isOpen}
+        onToggle={setIsOpen}
+        header={
+          <FormattedMessage id="xpack.ml.explorer.alertsPanel.header" defaultMessage="Alerts" />
+        }
+        headerItems={Object.entries(countByStatus ?? {}).map(([status, count]) => {
+          return (
+            <>
+              {status}{' '}
+              <EuiNotificationBadge size="m" color={count > 0 ? 'accent' : 'subdued'}>
+                {count}
+              </EuiNotificationBadge>
+            </>
+          );
+        })}
+      >
+        <div style={{ height: '60px', width: '100%', paddingLeft: `${Y_AXIS_LABEL_WIDTH}px` }}>
+          <Chart title={''} description={''}>
+            <Settings xDomain={annotationXDomain!} />
+            <LineSeries
+              id={'test'}
```

Comment:
Should id be `test` here?

Reply:
This chart has been removed in favour of a line date histogram powered by the Lens embeddable.